dev-ostelco-ios-clientTests/CoordinatorTests/RootCoordinatorTests.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  RootCoordinatorTests.swift
+//  dev-ostelco-ios-clientTests
+//
+//  Created by Ellen Shapiro on 6/5/19.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import ostelco_core
+@testable import Oya_Development_app
+import XCTest
+
+class RootCoordinatorTests: XCTestCase {
+    
+    lazy var testUserManager = MockUserManager()
+    
+    lazy var testCoordinator = RootCoordinator(window: UIWindow(),
+                                               userManager: self.testUserManager)
+    
+    override func setUp() {
+        super.setUp()
+        UserDefaultsWrapper.clearAll()
```

Comment:
What if we made it so we could feed a fresh fake user defaults in for each test? This way seems easy to mess up and get inconsistent test results.

Reply:
The wrapper would need some architectural changes to support this - I'd say maybe make a trello card to do it later.

Reply:
I'll make a note, agreed.